packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -108,6 +108,7 @@ export function Comments( {
 				justify="flex-start"
 				spacing="2"
 			>
+				<Text>{ __( 'Only logged in users can see Notes' ) }</Text>
 				{
```

Comment:
```suggestion
{
```

I don't think we need to add a message here, because the message `No comments available` will be removed in another PR.